projects/conobjects/Index.md

```diff
@@ -0,0 +1,9 @@
+--- 
+authors: 
+  - "Theodor Hillmann"
+authors-url: 
+  - "www.invitrocolor.com"
+layout: project
+title: "Connected Objects"
+type: visual fragment
```

Comment:
type should be `project`. I need this to sort out the projects in the pages index.